readme-eng.md

```diff
@@ -1,222 +1,321 @@
-   *  Ctrl+direction   attack, open, close, or dig
-* [Tools]
-   * r  Read a scroll
-   * q  Quaff a potion 
-   * u/Z  Use a staff
-   * a/z  Aim a wand
-   * z/a  Zap a rod
-   * E  Eat some food
-   * A  Activate an equipment
-   * F  Fuel your lantern/torch
-   * f/t  Fire a missile
-* [Action]
-   * d  Drop an item
-   * g  Pick up items
-   * R  Rest for a period
-   * o  Open a door or chest
-   * s  Search for traps/doors
-   * k/Ctrl+d  Destroy an item
-* [Magic]
-   * m  Cast a spell / use mental power
-   * G  Gain new spells/prayers
-   * U/O  Use bonus power
-   * b/P  Examine spells (choose each spell to get a description)
-* [Equipment] 
-   * w  Wear/wield equipment
-   * t  Take off equipment
-   * F  Fuel your lantern/torch
-* [Info] 
-   * x  Examine items in a store
-   * C  Character description
-   * I  Examine an *identified* item
-   * i  Inventory list
-   * l/x  Look around
-   * M  Full dungeon map
-   * ~  Display various information
-*  [Other]
-   * p  Command pets
-   * =  Set options
-   * $  Reload auto-picker preference file
-   * _  Edit auto-picker preference
-   * n/[  Repeat previous command
-   * @  Interact with macros
-*  [Help]
-   * ?  View the on-line help (Press a-z to choose a section,  space key to go next page.)
-
-
-## Inscriptions
-
-Below, only the new features in Hengband are explained.
-
-* You can inscribe on objects with "@ma" or '@'+<any command letter>+<any tag>. Then, type these alphabet letter in 'm' command to cast from the spellbook.
-* The inscriptions "@1" or '@'+<any tag number> make the objects able to be used by pressing <tag number> key in any command.  Any alphabet key cannot be used as a tag in this form.
-* You can inscribe on *identified* items with "%" or "%all". Then, "%" or "%all" is automatically replaced by strings which describe the item's abilities. 
-** Example:  You inscribe with "%all" on a pair of gauntlets of power which provide resistance to fire. The description will look like {St;Fi} which means bonus to Strength and resistance to fire. If you inscribe it with "%", its description will look like {Fi} which means its random added ability is resistance to fire.
-
-## Auto pickup/destroy
+### Install with `setgid`
 
-Hengband offers two different methods of auto-pickup/destroy.
+Run the following:
 
-### Powerful method
-* Hengband offers a very powerful auto-pickup/auto-destroy system which you may customize.  Press   *  '_' key to enter edit mode.  Please read the file "autopick_eng.txt" about the syntax.
-
-### Easy method
-* Since the powerful method is slightly difficult to use effectively, there is another very easy option. you can turn on 
-* the destroy_items option and/or the always_pickup option to use this method.  These options are located in (6) Easy Auto-Destroyer Options and (1) Input Options in the Set options ('=') command.
-
-## Tile graphics, Big tile, and Big screen
+```
+./configure --disable-japanes --with-setgid=games
```

Comment:
There's a typo here, it should be disable-japanese rather than disable-japanes .